common/views/themes/default.ts

```diff
@@ -8,7 +8,6 @@ import { inlineFonts } from './base/inline-fonts';
 import { layout } from './base/layout';
 import { normalize } from './base/normalize';
 import { row } from './base/row';
-import { transitions } from './base/transitions';
```

Comment:
Some of the view-transitions api code didn't get cleaned up when we decided to remove them apparently